config/example/config.txt

```diff
@@ -2,7 +2,7 @@
 # SERVERNAME spacestation13
 
 ## Server tagline: This appears on the hub entry.
-# SERVER_TAG_LINE The Perfect Mix of RP & Action
```

Comment:
RIP

Reply:
Are you reallyyyyy sure we should be changing this? This has been our tag line for literally years.